addons/PerformanceMonitor/Core/fn_recursiveInfoSending.sqf

```diff
@@ -24,7 +24,7 @@ if !(GVAR(FrameWindowStartNumber) isEqualTo 0) then {
 	private _passedTime = diag_tickTime - GVAR(FrameWindowStart);
 	
 	// calculate FPS and send to server
-	[PERF_INFO_RECEIVE, [[name player, "__Server"] select isServer, _numberOfFrames / _passedTime]] call CFUNC(serverEvent);
+	[PERF_INFO_RECEIVE, [[getPlayerUID player, "__Server"] select isServer, _numberOfFrames / _passedTime]] call CFUNC(serverEvent);

```

Comment:
I would still like to store the player's name as well. So imo we should store the UID in an extra column in the log-file and not _instead_ of the name.

Also with this change the log file gets slightly inconsistent since in that column you will now either find a UID or the String `__Server`. Thus creating a separate column for the UID and then using a UID of e.g. `0` (or leave it blank instead) to indicate the Server would probably be better :thinking:

Reply:
> I would still like to store the player's name as well. So imo we should store the UID in an extra column in the log-file and not _instead_ of the name.
> 
> Also with this change the log file gets slightly inconsistent since in that column you will now either find a UID or the String `__Server`. Thus creating a separate column for the UID and then using a UID of e.g. `0` (or leave it blank instead) to indicate the Server would probably be better 🤔

Good idea. Integrated.